src/components/table/TableItem.js

```diff
@@ -38,9 +39,24 @@ class TableItem extends PureComponent {
       editedCells: {},
       multilineText,
       multilineTextLines,
+      [this.props.rowId]: this.props,
     };
   }
 
+  shouldComponentUpdate(nextProps) {
+    if (
+      !_.isEqual(
+        _.omit(nextProps, 'dataHash'),
```

Comment:
You know that `dataHash` was deliberately added to know when data changed ?  :)

Reply:
Without excluding it you will always have re-render for each or the rows. I don't think the system was built to re-render all the table rows if only one is changing/selected.